fdbclient/TaskBucket.actor.cpp

```diff
@@ -170,19 +171,28 @@ class TaskBucketImpl {
 
 		// Get keyspace for the specified priority level
 		state Subspace space = taskBucket->getAvailableSpace(priority);
-
 		{
 			// Get a task key that is <= a random UID task key, if successful then return it
-			Key k = wait(tr->getKey(lastLessOrEqual(space.pack(uid)), Snapshot::True));
```

Comment:
getKey is translated to read range from "" to space.pack(uid). for blob restore case, reading range causes error. so use a specific getRange